src/components/Sections/Programs/tabs/Clips/List.js

```diff
@@ -9,7 +9,7 @@ const renderUnit = (unit, t) => {
   const { clips, relatedItems }
           = renderUnitHelper.commonRenderUnitForClips(unit, t);
 
-  return (
+  return unit ? (
```

Comment:
1. Try to return null instead if empty string. Will it work?
2. Change ?: to 
   if (!unit) {
      return null;
   }

   <jsx>...

Reply:
ok. it works

Reply:
null means for React: "do not render it"

Reply:
ok. good